otomoto/spiders/otomoto.py

```diff
@@ -37,30 +37,35 @@ def parse_car_page(self, response):
             'Marka pojazdu': 'brand',
             'Model pojazdu': 'model',
             'Rok produkcji': 'year',
+            'Wersja': 'version',
             'Przebieg': 'mileage',
             'Pojemność skokowa': 'capacity',
             'Moc': 'horse_power',
             'Rodzaj paliwa': 'fuel_type',
             'Skrzynia biegów': 'transmission',
             'Typ': 'type',
             'Liczba drzwi': 'number_of_doors',
+            'Kraj pochodzenia': 'origin_country',
             'Kolor': 'color',
             'Pierwszy właściciel': 'first_owner',
             'Bezwypadkowy': 'no_accidents',
             'Serwisowany w ASO': 'aso',
             'Stan': 'condition',
         }
         loader = OtomotoCarLoader(OtomotoItem(), response=response)
-        for params in response.css('.offer-params__item'):
 
+        for params in response.css('.offer-params__item'):
             property_name = params.css('.offer-params__label::text').extract_first().strip()
+            # print(property_name)
             if property_name in property_list_map:
-                css = params.css('div::text').extract_first().strip()
+                css = params.css('.offer-params__value::text').extract_first().strip()
+                print('Done')
                 if css == '':
                     css = params.css('a::text').extract_first().strip()
 
                 loader.add_value(property_list_map[property_name], css)
 
         loader.add_css('features', '.offer-features__item::text')
+        loader.add_css('price', '.offer-price__number::text')
```

Comment:
I think you need to normalize those values, because I am getting prices that looking like that:
```
    "price": "23 700        ",
    "price": "28 999        ",
    "price": "15 500        ",
    "price": "60 500        ",
    "price": "74 900        ",
    "price": "19 990        ",
    "price": "23 900        ",
```
while I would rather see them as
```"price:" 2500```

Reply:
I change te way to add price also I add price currency